src/pages/Welcome/style.css

```diff
@@ -1,18 +1,67 @@
+.welcome-container {
+  align-items: center;
+  background-color: #f2b807;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  padding: 20px;
+  width: 100%;
+}
+
 .welcome-form {
-  width: 250px;
+  max-width: 700px;
+  padding: 0 30px;
   text-align: left;
-  margin-top: 50px;
 }
 
 .welcome-text {
-  margin: 40px 0;
+  margin: 32px 0;
+  font-size: 18px;
+  font-family: 'Dosis', sans-serif;
```

Comment:
Will we use `Dosis` or `Montserrat` font? 🤔

Reply:
Yes, 🤣 my bad, I've already changed it